integration/raft/channel_participation_test.go

```diff
@@ -968,6 +1096,80 @@ func applicationChannelGenesisBlock(n *nwo.Network, orderers []*nwo.Orderer, p *
 	return genesisBlock
 }
 
+func systemChannelGenesisBlock(n *nwo.Network, orderers []*nwo.Orderer, peers []*nwo.Peer, channel string) *common.Block {
```

Comment:
This seems to be duplication of the existing logic for bootstrapping a network.  Did you look at simply re-using what's already present via nwo?

Reply:
The key difference is the flexibility of including different orderers in the consenters set without having a different configtx.yaml (or template). By default, nwo includes all orderers in the consenters set and the endpoints.